model/mainframe/registry_lpar.yaml

```
@@ -0,0 +1,11 @@
+groups:
+  - id: registry.mainframe.lpar
+    type: attribute_group
+    display_name: Mainframe LPAR attributes
+    brief: This document defines attributes of a Mainframe Logical Partition (LPAR).
+    attributes:
+      - id: mainframe.lpar.name
+        type: string
+        stability: experimental
```

nit: mind changing `experimental` to `development` in all new definitions?

```suggestion
        stability: development
```

we're moving towards more granular stability levels.